packages/design/src/colors.css

```diff
@@ -106,7 +108,8 @@
   --color-grey--lightest: rgb(244, 244, 244);
   --color-grey--dark: rgb(118, 118, 118);
 
-  --color-greyBlue: rgb(101, 120, 132);
+  --color--greyBlue--rgb: 101, 120, 132;
```

Comment:
Ooooh this kind of blew my mind, I think this is good! I had to read it a few times before I realized _how_ it was working, but this is, I think, a good solution.